utils/azureService.js

```diff
@@ -0,0 +1,15 @@
+const { DefaultAzureCredential } = require('@azure/identity');
+const { SecretClient } = require('@azure/keyvault-secrets');
+
+// const keyVaultName = process.env.MONGO_URL;
+const azureService = key => {
+  const KVUri = `https://horondi.vault.azure.net/secrets`;
```

Comment:
Please, give this variable mor explicit name, now it is really cryptographic

Reply:
It was copied from documentation. fixed